src/pudl/transform/params/ferc1.py

```diff
@@ -3507,6 +3557,83 @@
             },
         ],
     },
+    "accumulated_depreciation_plant_ferc1": {
+        "rename_columns_ferc1": {
+            "dbf": {
+                "columns": {
+                    "respondent_id": "utility_id_ferc1_dbf",
+                    "report_year": "report_year",
+                    "spplmnt_num": "spplmnt_num",
+                    "row_number": "row_number",
+                    "row_seq": "row_seq",
+                    "row_prvlg": "row_prvlg",
+                    "report_prd": "report_prd",
+                    "total_cde": "total_electric_plant",
+                    "future_plant": "future_electric_plant",
+                    "leased_plant": "leased_electric_plant",
+                    "electric_plant": "in_service_electric_plant",
+                    "xbrl_factoid": "ferc_account_label",
+                }
+            },
+            "duration_xbrl": {
+                "columns": {
+                    xbrl_col: f"{xbrl_col}_electric_plant"
```

Comment:
Were you going to pull out the simple "there's only one category" case into its own function? Did it already happen and I missed it, or did you decide not to, or is it still in the works?

Reply:
we haven't done this yet. I still think its a good idea but not the highest priority as this is a fine option for now.